diff --git a/external/f2fs-tools/Android.bp b/external/f2fs-tools/Android.bp
index 2b09807..6e13e25 100644
--- a/external/f2fs-tools/Android.bp
+++ b/external/f2fs-tools/Android.bp
@@ -211,6 +211,7 @@ cc_binary {
         "fsck_main_src_files",
     ],
     host_supported: true,
+    vendor_ramdisk_available: true,
     cflags: ["-DWITH_RESIZE", "-DWITH_DEFRAG", "-DWITH_DUMP"],
     srcs: ["fsck/fsck.c", "fsck/resize.c", "fsck/defrag.c"],
     shared_libs: [
